build-scripts/windist.ps1

```diff
@@ -0,0 +1,13 @@
+rm -Recurse -Force bindist
+
+mkdir bindist
+cp Cataclysm-vcpkg-static-Release-x64.exe bindist/cataclysm-tiles.exe
+cp tools/format/JsonFormatter-vcpkg-static-Release-x64.exe bindist/json_formatter.exe
+
+mkdir bindist/lang
+cp -r lang/mo bindist/lang
+
+$extras = "data", "doc", "gfx", "LICENSE.txt", "LICENSE-OFL-Terminus-Font.txt", "README.md", "VERSION.txt"
```

Comment:
I suggest excluding `data/mods/TEST_DATA` from release distribution.

Reply:
That's a good idea, but the official distro also includes this data and for now at least I want to try to do as faithful of an adaptation as possible.

Reply:
I'm initially trying to just match the existing distribution one for one, which does not remove the test data.